package.json

```
@@ -22,6 +22,14 @@
   },
   "license": "MIT",
   "author": "stylelint",
+  "exports": {
+    ".": {
+      "import": "./lib/index.mjs",
+      "require": "./lib/index.cjs"
+    },
+    "./package.json": "./package.json",
+    "./lib/utils/*": "./lib/utils/*"
```

[suggest] If we returned to `.js` from `.mjs` in the next major version (pure ESM), avoiding `.mjs` access might be better for compatibility. Any thoughts?

```suggestion
    "./lib/utils/*": {
      "import": "./lib/utils/*.mjs",
      "require": "./lib/utils/*.cjs"
    }
```

---

We realize importing without an extension is unnecessary because we will disallow access to internal utils in the next major release.

Also, `import` without an extension doesn't seem to fit the ESM spec.

See https://github.com/stylelint/stylelint/pull/7307#issuecomment-1801789503